meson.build

```diff
@@ -98,12 +85,6 @@ conf.set10('HAVE_SPLIT_BIN', split_bin,
            description : 'bin and sbin directories are separate')
 
 rootprefixdir = get_option('rootprefix')
-# Unusual rootprefixdir values are used by some distros
-# (see https://github.com/systemd/systemd/pull/7461).
-rootprefix_default = split_usr ? '/' : '/usr'
-if rootprefixdir == ''
-        rootprefixdir = rootprefix_default
-endif
```

Comment:
The setting of `rootprefixdir` should stay. This is independent of the usr split.

Reply:
I've simply changed it to have a default in meson_options to 'usr', isn't that enough?